### PR TITLE
Log warnings on long i18n message ids

### PR DESCRIPTION
**Pull request by toolness**
From a Slack conversation:

> So right now after merging our two i18n PRs it looks like the norent i18n bundle for English at https://tenants2-staticfiles-atul-dev.s3.amazonaws.com/frontend/locales-en-norent-chunk.bb81f08785cb61c11a68.bundle.js is about 9k gzipped, which is a bit of a bummer since, being the english catalog, it's literally all just an object full of strings like `"Your password": "Your password"`.  And then our source bundle has a bunch of references to those strings (since they need to be looked up in the 18n bundle), so when the ID strings consist of entire paragraphs of text, it's like we've got 3 separate instances of them in our source code, which probably bloats it when it's all added up.

This adds functionality to our localebuilder tool which ~~logs warnings (which we could convert to errors in a future PR)~~ raises errors whenever lingui message catalog IDs are longer than ~~100~~ 175 characters, indicating that significant space might be saved if we shorten them.  We can always change the limit if we want, too.

Here's a screenshot of its output:

> image

Since localebuilder is run in production and the output is colorized, this PR also moves `chalk` from our `devDependencies` to `dependencies` (it's already ultimately in there anyways due to various third-party packages in `dependencies` already needing it).

### CODE REVIEW

**Comment by sraby** [comment]
Ok @toolness, I just added ID tags to fix all of our msgid-length warnings! I decided to set the threshold to 175 characters, as this seemed to be a good break point in the list of messages that we got.

File: frontend/lib/norent/components/social-icons.tsx [hunk 1]
```diff
@@ -17,15 +17,19 @@ const socialMediaPageLinks = [
 
 const prefilledTwitterCopy = () =>
   li18n._(
-    t`No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. This free tool sends a certified letter informing them of your protections. Join the #cancelrent movement at norent.org.`
+    t(
+      "norent.tweetTemplateForSharingNoRent"
+    )`No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. This free tool sends a certified letter informing them of your protections. Join the #cancelrent movement at norent.org.`
```
Comment (toolness) [inline frontend/lib/norent/components/social-icons.tsx:22]: Oh super dope, I didn't realize there was a way to manually specify an ID for these non-JSX thingys!

**Comment by toolness** [comment]
Awesome thanks!  I'll fix up the letter builder ones and then activate HARDFAIL and merge!

**Comment by toolness** [comment]
oh wow, thanks, I didn't actually realize you addressed _all_ the warnings, not just the ones in primary pages!

**Pull request merged**